smart-hadoop-support/smart-hadoop/src/main/java/org/smartdata/hdfs/metric/fetcher/NamespaceFetcher.java

```diff
@@ -286,6 +286,8 @@ public void run() {
           if (startAfter != null) {
             pendingParent = parent;
           }
+        } else {
+          LOG.warn("Configured directory does not exist");
```

Comment:
Should not tackle unexisted dir in this section..
It seems that startAfter == null doesn't mean that the configured directory does not exist.